.github/workflows/auto-assign-project.yml

```
@@ -1,19 +1,34 @@
-    name: Assign to One Project
+
     steps:
-    - name: Assign NEW issues and NEW pull requests to project 2
-      uses: srggrs/assign-one-project-github-action@1.2.1
-      if: github.event.action == 'opened'
-      with:
-        project: 'https://github.com/users/minhnt3/projects/1'
-        column_name: 'No Status'
\ No newline at end of file
+      - name: 🔍 Check PR title
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)